Find a functionally equivalent code:android.media.MediaFocusControl.onRcDisplayClear:COMMENT
Method Modifier: private     
Comment:/**
 * Called when processing MSG_RCDISPLAY_CLEAR event
 */

@@ -1,7 +1,7 @@
 {
     if (DEBUG_RC)
         Log.i(TAG, "Clear remote control display");
-    synchronized (mRCStack) {
+    synchronized (mPRStack) {
         synchronized (mCurrentRcLock) {
             mCurrentRcClientGen++;
             // synchronously update the displays and clients with the new client generation

